src/main/boilerplate/spray/json/ProductFormatsInstances.scala.template

```diff
@@ -29,7 +31,7 @@ trait ProductFormatsInstances { self: ProductFormats with StandardFormats =>
       fields.sizeHint(1 * 2)
       [#fields ++= productElement##2Field[P1](fieldName1, p, 0)#
       ]
-      JsObject(fields: _*)
+      JsObject(fields.toSeq: _*)
```

Comment:
varargs parameter type is `collection.immutble.Seq` instead of `collection.Seq` since Scala 2.13.0-M4